composer.json

```diff
@@ -10,13 +10,17 @@
             "email": "mailchimp@ebizmarts-desk.zendesk.com"
         }
     ],
-    "require": {
-        "magento-hackathon/magento-composer-installer": "*"
+    "suggest": {
+        "magento-hackathon/magento-composer-installer": "Map/copy module files with your Magento installation"
+    },
+    "require-dev": {
+        "magento-ecg/coding-standard": "^2.0",
+        "magento/marketplace-eqp": "^1.0"
     },
     "repositories": [
         {
-            "type": "composer",
```

Comment:
The package `magento/marketplace-eqp` in the repo `packages.firegento.com` links to a archive hosted at `repo.magento.com`. Magento requires you to fill in authentication information to use this repo. ([link to docs](http://devdocs.magento.com/guides/v2.0/install-gde/prereq/connect-auth.html))
I want to make sure that `magento/marketplace-eqp` is loaded from GitHub since it only need a GitHub Token and is much more common to have a GitHub account then a magento.com account.

Also I could not find any written reason in ed7ff245b1817463af0c161edc74dab12f23456a (the commit that introduced the change) for the `packages.firegento.com` repo being added